sql/moz-fx-data-shared-prod/telemetry_derived/desktop_funnel_activation_v1/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+description: >
+  Measures the number of activated new profiles in the Firefox desktop
+  funnel
+friendly_name: Firefox Desktop Funnel Activation
+labels:
+  incremental: true
+owners:
+  - ascholtz@mozilla.com
+scheduling:
+  dag_name: bqetl_desktop_funnel
+  parameters: [
+    # Set the submission_date to the date from 8 days ago
+    'submission_date:DATE:{{ macros.ds_format((execution_date + macros.timedelta(days=-8)).to_datetime_string(), "%Y-%m-%d %H:%M:%S", "%Y-%m-%d") }}'   # yamllint disable-line rule:line-length
+  ]
+  date_partition_parameter: null
+  destination_table: desktop_funnel_activation_v1${{ macros.ds_format((execution_date + macros.timedelta(days=-8)).to_datetime_string(), "%Y-%m-%d %H:%M:%S", "%Y%m%d") }}   # yamllint disable-line rule:line-length
```

Comment:
I changed the logic to be backward-looking. This way macros for the `submission_date` parameter aren't needed.
Since we won't have user-facing views, I kept the parameter for the table partition.